### ISSUE TITLE
"referraling" inline docs typo

### ISSUE THREAD

**Issue by DrewAPicture** [AffiliateWP/AffiliateWP#1315]
There's a typo in the DocBlock for `Affiliate_WP_MemberPress::add_pending_referral()`:
```php
	/**
	 * Store a pending referraling when a one-time product is purchased
	 *
	 * @access  public
	 * @since   1.5
	*/
	public function add_pending_referral( $txn ) {
```
PR: #1282

**Comment by pippinsplugins** [comment]
That's for when I'm tried of referring customers and would prefer to be a referral'er that spends his days referraling.

**Comment by DrewAPicture** [comment]
Merged #1282
